runelite-client/src/main/java/net/runelite/client/ui/overlay/worldmap/WorldMapOverlayMouseListener.java

```diff
@@ -68,8 +69,18 @@ public MouseEvent mousePressed(MouseEvent e)
 				{
 					if (worldMapPoint.isJumpOnClick())
 					{
+						WorldPoint target;
+						if (worldMapPoint instanceof TransportationPoint)
```

Comment:
This is wrong. You should not use class from plugin in core package. Just use getWorldPoint

Reply:
getWorldPoint won't work I think:
![image](https://user-images.githubusercontent.com/10108473/63224211-309daf00-c1c1-11e9-8e8d-1e6388913b78.png)

I will push another solution later today.

Reply:
The probably just extend the worldMapPoint api

Reply:
Pushed the changes.

Now that I think about it, I should have probably just changed the jumpOnClick from boolean to WorldPoint, that way it could omit the target null-checking in the mouse listener. But I may be overthinking it, so I'm not changing anything until the review.